docs/mpcdf.md

```diff
@@ -12,24 +12,38 @@ All profiles use `singularity` as the corresponding containerEngine. To prevent
 
 >NB: Nextflow will need to submit the jobs via SLURM to the clusters and as such the commands above will have to be executed on one of the head nodes. Check the [MPCDF documentation](https://www.mpcdf.mpg.de/services/computing).
 
-## cobra
+## Global Profiles
 
-To use: `-profile cobra,mpcdf`
+### cobra
+
+To use: `-profile mpcdf,cobra`
 
 Sets the following parameters:
 
 - Maximum parallel running jobs: 8
-- Max. memory: 750.GB
-- Max. CPUs: 80
+- Max. memory: 725.GB
+- Max. CPUs: 40
 - Max. walltime: 24.h
 
-## raven
+### raven
 
-To use: `-profile raven,mpcdf`
+To use: `-profile mpcdf,raven`
 
 Sets the following parameters:
 
 - Maximum parallel running jobs: 8
-- Max. memory: 368.GB
-- Max. CPUs: 192
+- Max. memory: 200000.MB (2.TB)
```

Comment:
This is 0.2TB ?

Reply:
Why not writing 2000.GB? to make it easier to read?

Reply:
For clarity - we specify with MB as apparently there was some wierd overhead that's not included and 2TB doesn't include that overhead (or something), but 2000000.MB is low enough away from a full 2TB to work

:shrug: it's what the MPCDF admins recommended to @maltesemike

Reply:
🤦🏻 Fair enough then,